frontend/css/theme/bootstrap/utilities.less

```diff
@@ -27,9 +27,6 @@
 .hide {
   display: none !important;
 }
-.show {
-  display: block !important;
-}
```

Comment:
Is this definitely safe to remove?

Reply:
Hm, let's see if this causes any issues with it missing.
If it does, I'll suggest to reinstate .show and (contrary to everything I said so far) change the side-nav to add important 
 : `display: flex !important;`
 
 Let's see.